### PR TITLE
add new lab-nxp lab

### PR DESCRIPTION
**Pull request by khilman**
Add NXP lab and initial ls2088-rdb board

As requested by:
- Suram Suram <suram@nxp.com>
- Philippe Mazet <philippe.mazet@nxp.com>

Signed-off-by: Kevin Hilman <khilman@baylibre.com>

### CODE REVIEW

File: test-configs.yaml [hunk 1]
```diff
@@ -749,6 +749,11 @@ device_types:
             - 'defconfig+CONFIG_RAMDOMIZE_BASE=y'
           kernel: ['v3.']
 
+  fsl-ls2088a-rdb:
```
Comment (gctucker) [inline test-configs.yaml]: Please follow the alphabetical order of the devices.

File: lab-configs.yaml [hunk 2]
```diff
@@ -122,3 +122,7 @@ labs:
             - baseline
             - boot
             - kselftest
+
+  lab-nxp:
+    lab_type: lava
+    url: 'http://59.144.98.45/RPC2/'
```
Comment (gctucker) [inline lab-configs.yaml]: I think this should really be `https` otherwise some API tokens are going to be sent in clear.

Comment (gctucker) [reply lab-configs.yaml]: Of course with https you probably also need a domain name...

File: test-configs.yaml (hunk 1 above)
Comment (khilman) [reply test-configs.yaml]: oops, it's alphabetical for NXP.  ;)   Will fix.

File: lab-configs.yaml (hunk 2 above)
Comment (khilman) [reply lab-configs.yaml]: Ack.  I've requested an https URL + domain-name from NXP.

Comment (gctucker) [reply lab-configs.yaml]: Also it should be between `lab-mhart` and `lab-pengutronix`.

Comment (gctucker) [reply lab-configs.yaml]: @khilman Any news about a domain name for this lab?

Comment (khilman) [reply lab-configs.yaml]: No news.  I'll ping NXP.

Comment (khilman) [reply lab-configs.yaml]: Domain name and HTTPS URL now in place.  Updated.

**Comment by khilman** [comment]
Example LAVA job, created/submitted with kci_test:
https://lavalab.nxp.com/scheduler/job/3669

File: lab-configs.yaml (hunk 2 above)
Comment (gctucker) [reply lab-configs.yaml]: Great news!

**Comment by gctucker** [comment]
Great.  I see you've removed the "staging-skip" label, that's fine but it won't work on staging.kernelci.org until all the API tokens have been created and distributed to the correct places.

**Comment by khilman** [comment]
Tokens were created/distributed shortly after submitting PR.

**Comment by gctucker** [comment]
> Tokens were created/distributed shortly after submitting PR.

Erm, the LAVA API token `lab-nxp-lava-api` is missing:
https://staging-ci.kernelci.org/job/kernel-build-trigger/538/console

**Comment by gctucker** [comment]
OK it's working now with the new staging jenkins instance:
https://staging.kernelci.org/boot/id/5e42ab87c5982d8996e43388/

**Comment by gctucker** [comment]
Tested OK again on staging:
https://staging.kernelci.org/test/job/kernelci/branch/staging.kernelci.org/kernel/staging-20200225.0/plan/baseline/

**Pull request merged**